plugins/analytics/public/virustotal_api.py

```diff
@@ -80,57 +82,66 @@ def analyze(observable, results):
         result = {'raw': json_string}
 
         if isinstance(observable, Ip):
-            # Parse results for ip
-            if json_result.get('as_owner'):
-                result['Owner'] = json_result['as_owner']
-                o_isp = Company.get_or_create(name=json_result['as_owner'])
-                links.update(
-                    observable.active_link_to(
-                        o_isp, 'hosting', 'virustotal_query'))
-
-            if json_result.get('detected_urls'):
-                result['detected_urls'] = json_result['detected_urls']
-                for detected_url in json_result['detected_urls']:
-                    o_url = Url.get_or_create(value=detected_url['url'])
+            if json_result['response_code'] == 1:

```

Comment:
You can also do the reverse logic to save an indentation: 

```
if json_result['response_code'] !- 1:
  return # Or do whatever it is you do when you get an invalid response
```

Additionally, this guard can also go before all the `isinstance` checks.